attributes/default.rb

```diff
@@ -63,6 +63,8 @@
 default['hopsworks']['http_logs']['enabled']     = "true"
 default['hopsworks']['env_var_file']             = "#{node['hopsworks']['domains_dir']}/#{node['hopsworks']['domain_name']}_environment_variables"
 
+default['glassfish']['reschedule_failed_timer']     = "true"
```

Comment:
@robzor92 nitpick, shouldn't it be plural? `timers`

Reply:
I used the same naming scheme they do in the asadmin command, is it ok?

Reply:
of course, it seems more logical to me to have it in plural but lets stick to what they do :)